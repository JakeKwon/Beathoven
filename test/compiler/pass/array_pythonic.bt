int [] arr1 = [9, 8, 7];
int [] arr2 = [6, 5, 4];
int [] a= [arr1, arr2, 3, 4, 5];
print("printing joined array element: ", a[3], "\n");
a[3] = 300;
print("printing edited joined array element: ", a[3], " \n");
int [] b = a[1:4];
print("printing sub array element: ", b[0], " \n");
