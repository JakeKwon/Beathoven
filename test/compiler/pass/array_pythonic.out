printing joined array element: 6
printing edited joined array element: 300
printing sub array element: 8